scripts/master.xml

```diff
@@ -1,888 +1,952 @@
+						<in_model_definition>Deep learning model package (.dlpk)</in_model_definition>
+						<arguments>The function arguments are defined in the Python raster function class. </arguments>
+				</DetectChangeUsingDeepLearning>
+				<!-- DOUDL Detect Objects Using Deep Learning  -->
+				<!--https://pro.arcgis.com/en/pro-app/latest/tool-reference/image-analyst/detect-objects-using-deep-learning.htm-->
+				<DetectObjectsUsingDeepLearning>
+						<in_raster>The input image that will be used to detect objects.</in_raster>
+						<out_detected_objects>The output feature class that will contain geometries circling the object or objects detected in the input image.</out_detected_objects>
+						<in_model_definition> A deep learning model package (.dlpk)</in_model_definition>
+						<model_arguments>The function arguments defined in the Python raster function class</model_arguments>
+						<run_nms>NO_NMS;NMS</run_nms>
+						<confidence_score_field>The name of the field in the feature class that will contain the confidence scores as output by the object detection method.</confidence_score_field>
+						<class_value_field>The name of the field that will contain the class or category label in the output feature class.</class_value_field>
+						<max_overlap_ratio>The maximum overlap ratio for two overlapping features, which is defined as the ratio of intersection area over union area. The default is 0.</max_overlap_ratio>
+						<processing_mode>PROCESS_AS_MOSAICKED_IMAGE;PROCESS_ITEMS_SEPARATELY</processing_mode>
+				</DetectObjectsUsingDeepLearning>
+				<!-- NMS NON Maximum Suppression  -->
+				<!--https://pro.arcgis.com/en/pro-app/latest/tool-reference/image-analyst/non-maximum-suppression.htm-->
+				<NonMaximumSuppression>
+						<in_featureclass>The input feature class or feature layer containing overlapping or duplicate features.</in_featureclass>
+						<confidence_score_field>The field in the feature class that contains the confidence scores as output by the object detection method.</confidence_score_field>
+						<out_featureclass> The output feature class with the duplicate features removed.</out_featureclass>
+						<class_value_field>The name of the field that will contain the class or category label in the output feature class.</class_value_field>
+						<model_arguments>The function arguments defined in the Python raster function class</model_arguments>
+						<max_overlap_ratio>The maximum overlap ratio for two overlapping features. This is defined as the ratio of intersection area over union area. The default is 0.</max_overlap_ratio>
+				</NonMaximumSuppression>
+				<!-- CF Compute Fiducials (Data Management)  -->
+				<!-- https://pro.arcgis.com/en/pro-app/latest/tool-reference/data-management/compute-fiducials.htm -->
+				<ComputeFiducials>
+					<out_fiducial_table>#</out_fiducial_table>
+					<where_clause>#</where_clause>
+                    <fiducial_templates>#</fiducial_templates>#
```

Comment:
Delete the additional hash

Reply:
@vijaygit02 Yes, although it does not make the XML invalid, we should remove that, thanks!